lua/better_escape.lua

```diff
@@ -49,73 +51,76 @@ local function clear_mappings()
 end
 
 -- WIP: move this into recorder.lua ?
-local last_key = nil
+-- When a first_key is pressed, `recorded_key` is set to it 
+-- (e.g. if jk is a mapping, when 'j' is pressed, `recorded_key` is set to 'j')
+local recorded_key = nil 
 local bufmodified = false
```

Comment:
perhaps initial value nil?

Reply:
The initial value is nil ?

Reply:
I'm talking about the bufmodified
perhaps a bit confusing because github wants to be smart and displays things with context
but imo we shouldn't set an initial value for it if we actually don't know it
if anything directly make it `vim.bo.bufmodified`

Reply:
> but imo we shouldn't set an initial value for it if we actually don't know it
 
Sure.